docs/index.md

```diff
@@ -52,7 +52,7 @@ The distance from Ideal Bar (1) to Starkweather Brewing (9) is about 0.5 miles.
 | 1. Ideal Bar | Cash only | No | Very dive-y |
```

Comment:
Should consider re-ordering the MyMaps in google so the order follows the tentative order here.